### DISCUSSION TITLE
How about including Lepton theme?

### DISCUSSION THREAD

**Discussion by black7375** [pulse-browser/browser#112] [Ideas]
Hello, I'am author of [Lepton custom theme](https://github.com/black7375/Firefox-UI-Fix).
I accidentally found your work and looks fun.

What do you think of containing my theme in your browser?
It is the most popular custom theme used by quite a few people, and is also used in various forks such as [Warterfox](https://www.waterfox.net/)([G4](https://www.waterfox.net/docs/releases/g4.0.0), [G5](https://www.waterfox.net/docs/releases/G5.0b2)) and [Floorp browser](https://blog.ablaze.one/1816/2022-05-12/), [RunningCheese](https://www.runningcheese.com/firefox).

**Comment by trickypr** [comment]
I am going to start looking into this today, but it will take some time. I may need to make some pull requests to your upstream repo to get things working the way I want them to

**Reply by trickypr** [reply]
This didn't make it in before the feature freeze, so it will not be included in the next stable release. You can follow progress on the tracking issue: https://github.com/pulse-browser/browser/issues/117
